modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -76,11 +76,72 @@ data:
     </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
 ----
 
+The following example shows two containers that reference image streams that are running in the `openshift` namespace. One container handles the JNLP contract for launching Pods as Jenkins Agents. The other container uses an image with tools for building code in a particular coding language:
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: jenkins-agent
+  labels:
+    role: jenkins-slave
```

Comment:
I am bit confused here with labels `role: jenkins-slave`. AFAIK `jenkins-slave` is legacy one and IIUC it should be `jenkins-agent`, no? 

@akram please correct me if I am wrong.

Reply:
@prietyc123 you are correct. We should use the `jenkins-agent` terminology now even though internally, the legacy label is still handled.

Reply:
Updated. Thanks. I've also verified that there are no other instances of "jenkins slave" or "jenkins-slave" in the documentation.